src/styles/themes/dark.scss

```diff
@@ -1,48 +1,46 @@
 // Vars -------------------------------------------------------------- //
-@import '../vars';
+@import "../vars";
 
 // Dark Theme ------------------------------------------------------- //
 .basicContext {
-
-	padding: 6px 0;
-	background-color: rgba(40, 40, 40, 0.98);
-	box-shadow: 0 2px 15px rgba(0, 0, 0, .4), 0 0 0 1px #222;
-	border-radius: 3px;
-
-	&__item {
-		margin-bottom: 2px;
-
-		&--separator {
-			margin: 4px 0;
-			background-color: #222;
-		}
-
-		&--disabled {
-			opacity: .5;
-		}
-
-		&:last-child {
-			margin-bottom: 0;
-		}
-	}
-
-	&__data {
-		padding: 6px 14px;
-		color: white;
-	}
-
-	&__item:not(.basicContext__item--disabled):hover &__data {
-		background-color: $basicContext__color;
-	}
-
-	&__item:not(.basicContext__item--disabled):active &__data {
-		background-color: darken($basicContext__color, 10%);
-	}
-
-	&__icon {
-		margin-right: 8px;
-		width: 12px;
-		text-align: center;
-	}
-
-}
\ No newline at end of file
+  padding: 6px 0;
```

Comment:
If we are already going to change the indention and whitespace everywhere, we should at least also use tabulators for indention as we do everywhere else.